src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```
@@ -21,13 +21,13 @@
   --#{$clipboard-copy}__expandable-content--OutlineOffset: var(--pf-t--global--spacer--xs);
 
   // Group
-  --#{$clipboard-copy}__group--Gap: var(--pf-t--global--spacer--xs);
+  --#{$clipboard-copy}__group--Gap: var(--pf-t--global--spacer--gap--control-to-control--default);
 
   // Inline
   --#{$clipboard-copy}--m-inline--PaddingInlineStart: var(--pf-t--global--spacer--xs);
   --#{$clipboard-copy}--m-inline--PaddingInlineEnd: var(--pf-t--global--spacer--xs);
   --#{$clipboard-copy}--m-inline--BackgroundColor: var(--pf-t--global--background--color--secondary--default);
-  --#{$clipboard-copy}__actions--Gap: var(--pf-t--global--spacer--sm);
+  --#{$clipboard-copy}__actions--Gap: var(--pf-t--global--spacer--gap--action-to-action--default);
   --#{$clipboard-copy}__actions--MarginInlineStart: var(--pf-t--global--spacer--xs);
```

Confirmed with @lboehling we can use the new var you created `--pf-t--global--spacer--gap--text-to-element--compact` here

```suggestion
  --#{$clipboard-copy}__actions--MarginInlineStart: var(--pf-t--global--spacer--gap--text-to-element--compact);
```

---

Just to confirm, did you mean update teh MarginInlineStart token to the text-to-element spacer, or update the actions--Gap?